.github/workflows/run-tests.yml

```diff
@@ -21,9 +21,10 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
           extensions: fileinfo
+          coverage: xdebug
       - name: Install Hyde
         run: git clone -b master https://github.com/hydephp/hyde.git
-   
+
```

Comment:
What if we explicitly create the output directories before running the tests? A simple mkdir should do. While it won't work on Windows it should still help with debugging.

Reply:
I think the directories are in `hyde/build/coverage` as opposed to `build/coverage`

Reply:
Oh yeah, that's right. They probably are. Since we are running the tests through the hyde/hyde package.